plugins/azure/applicationGateway/agMaxRequestBodySize.js

```diff
@@ -0,0 +1,66 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Application Gateway Max Request Body',
```

Comment:
```suggestion
    title: 'Application Gateway Request Body Size',
```
I think name should be like this